packages/components/src/shortcuts/Shortcut.ts

```diff
@@ -92,6 +92,7 @@ export enum KEY {
   F10 = 'F10',
   F11 = 'F11',
   F12 = 'F12',
+  LEFT_CLICK = 'Left Click',
```

Comment:
Left click isn't really a key from the keyboard, so I don't think this should be here.
I don't think we show anything showing mouse click shortcuts in the context menu - @dsmmcken should we bother showing a shortcut for this action?